scripts/generate-full-corvid-types.js

```diff
@@ -1,48 +1,13 @@
 #!/usr/bin/env node
-const tmp = require("tmp");
 const fs = require("fs-extra");
-const path = require("path");
-const ts = require("typescript");
 const without_ = require("lodash/without");
+const createTsProgram = require("./util").createTsProgram;
```

Comment:
Before I go through this file, can you explain the change in this file in a few words?
Supposedly, from reading the title of this PR, this file is unrelated. Is it an unrelated refactor?

In general, separate unrelated changes/concepts to different commits/PRs. Each commit should do one thing and one thing only. Makes it easier to review, and also easier to roll back changes in case it is needed (we are human, and we all make mistakes sometimes)

Reply:
I is very needed for this change, I wanted to use the same logic of the using the tsCompiler so I've moved the more general stuff to a separate file.

The idea is that both generating the module list and the typings file require starting a ts program.